RemoteIDModule/transport.cpp

```diff
@@ -43,9 +49,10 @@ uint8_t Transport::arm_status_check(const char *&reason)
         reason = "missing location message";
     } else if (!g.have_basic_id_info() && (last_basic_id_ms == 0 || now_ms - last_basic_id_ms > max_age_other_ms)) {
         reason = "missing basic_id message";
-    } else if (last_self_id_ms == 0  || now_ms - last_self_id_ms > max_age_other_ms) {
-        reason = "missing self_id message";
-    } else if (last_operator_id_ms == 0 || now_ms - last_operator_id_ms > max_age_other_ms) {
+    //SelfID message is not mandatory.
+    //} else if (last_self_id_ms == 0  || now_ms - last_self_id_ms > max_age_other_ms) {
+    //    reason = "missing self_id message";
+    } else if ((last_operator_id_ms == 0 || now_ms - last_operator_id_ms > max_age_other_ms) && g.region == 3) {
```

Comment:
this looks like we don't make operator_id mandatory in the US? I thought it was mandatory?

Reply:
The Operator ID is only mandatory in the EU.
See a summary of the various fields here: https://github.com/opendroneid/opendroneid-core-c#comparison

Reply:
See comment @friissoren